app/assets/stylesheets/_base.scss

```diff
@@ -13,6 +13,16 @@
   h1 { @include h1; }
   h2 { @include h2; }
 
+  section { 
+    color: inherit; //can remove when old stylesheets are removed in refresh
+    font-family: inherit; //can remove when old stylesheets are removed in refresh
+  } 
+
+  div { 
+    color: inherit; //can remove when old stylesheets are removed in refresh
+    font-family: inherit; //can remove when old stylesheets are removed in refresh
+  }
+
   p,a { color: inherit; } //can remove when old stylesheets are removed in refresh
   
   a:link, //override old stylesheets - can removed when refresh is done
```

Comment:
Shall we use a consistent comment for this so that it's easy to find and delete when that happens?